packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -4,71 +4,35 @@
-import {ShopperPromotions, ShopperPromotionsTypes} from 'commerce-sdk-isomorphic'
-import {ShopperSearch, ShopperSearchTypes} from 'commerce-sdk-isomorphic'
+import {ShopperBaskets} from 'commerce-sdk-isomorphic'
+import {ShopperContexts} from 'commerce-sdk-isomorphic'
+import {ShopperCustomers} from 'commerce-sdk-isomorphic'
+import {ShopperDiscoverySearch} from 'commerce-sdk-isomorphic'
+import {ShopperGiftCertificates} from 'commerce-sdk-isomorphic'
+import {ShopperLogin} from 'commerce-sdk-isomorphic'
+import {ShopperOrders} from 'commerce-sdk-isomorphic'
+import {ShopperProducts} from 'commerce-sdk-isomorphic'
+import {ShopperPromotions} from 'commerce-sdk-isomorphic'
+import {ShopperSearch} from 'commerce-sdk-isomorphic'
 
-export * from './ShopperBaskets/types'
-export * from './ShopperCustomers/types'
-export * from './ShopperOrders/types'
-export * from './ShopperProducts/types'
-export * from './ShopperPromotions/types'
-export * from './ShopperSearch/types'
-
-export type ShopperBasketsInstance = ShopperBaskets<
-    ShopperBasketsTypes.ShopperBasketsParameters & Record<string, unknown>
->
-export type ShopperContextsInstance = ShopperContexts<
-    ShopperContextsTypes.ShopperContextsParameters & Record<string, unknown>
->
-export type ShopperCustomersInstance = ShopperCustomers<
-    ShopperCustomersTypes.ShopperCustomersParameters & Record<string, unknown>
->
-export type ShopperDiscoverySearchInstance = ShopperDiscoverySearch<
-    ShopperDiscoverySearchTypes.ShopperDiscoverySearchParameters & Record<string, unknown>
->
-export type ShopperGiftCertificatesInstance = ShopperGiftCertificates<
-    ShopperGiftCertificatesTypes.ShopperGiftCertificatesParameters & Record<string, unknown>
->
-export type ShopperLoginInstance = ShopperLogin<
-    ShopperLoginTypes.ShopperLoginParameters & Record<string, unknown>
->
-export type ShopperOrdersInstance = ShopperOrders<
-    ShopperOrdersTypes.ShopperOrdersParameters & Record<string, unknown>
->
-export type ShopperProductsInstance = ShopperProducts<
-    ShopperProductsTypes.ShopperProductsParameters & Record<string, unknown>
->
-export type ShopperPromotionsInstance = ShopperPromotions<
-    ShopperPromotionsTypes.ShopperPromotionsParameters & Record<string, unknown>
->
-export type ShopperSearchInstance = ShopperSearch<
-    ShopperSearchTypes.ShopperSearchParameters & Record<string, unknown>
->
-
-export interface ApiClients {
-    shopperBaskets: ShopperBasketsInstance
-    shopperContexts: ShopperContextsInstance
-    shopperCustomers: ShopperCustomersInstance
-    shopperDiscoverySearch: ShopperDiscoverySearchInstance
-    shopperGiftCertificates: ShopperGiftCertificatesInstance
-    shopperLogin: ShopperLoginInstance
-    shopperOrders: ShopperOrdersInstance
-    shopperProducts: ShopperProductsInstance
-    shopperPromotions: ShopperPromotionsInstance
-    shopperSearch: ShopperSearchInstance
+type CommerceApiProviderParams = {
```

Comment:
How about renaming this type? Because there are additional things found in the params of the Provider. See: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/commerce-api-provider/packages/commerce-sdk-react/src/provider.tsx#L23-L28

```suggestion
type ApiClientConfigParams = {
```